provisioning/datasources/datasources.yml

```diff
@@ -18,3 +18,19 @@ datasources:
     defaultEditor: builder
     sigV4Auth: true
     sigV4AuthType: keys
+    sigV4ExternalId: asf
+  secureJsonFields:
+    sigV4AccessKey: "aaa"
```

Comment:
I think we can replace this with accessKey and secretKey, it should work correctly fill out the data then. ALso, do we need accessControl below for provisioning?